### ISSUE TITLE
mangadex updates - Error - could not retrieve manga title

### ISSUE THREAD

**Issue by orbea** [frozenpandaman/mangadex-dl#37]
mangadex-dl: https://github.com/frozenpandaman/mangadex-dl/commit/3883aa49d52e2c7c3f914f43a6e5fdd3aeebbedf

It seems mangadex made some changes that broke the script again.
```
$ ./mangadex-dl.py 
mangadex-dl v0.5.0
Enter manga URL or ID: https://mangadex.org/title/077a3fed-1634-424f-be7a-9a96b7f07b78/kingdom
Error - could not retrieve manga title
```

It seems to be the same result with any series.

**Comment by ArjixWasTaken** [comment]
It's because when you use a url the script splits the url on "/"
This makes
 ```python
"https://mangadex.org/title/077a3fed-1634-424f-be7a-9a96b7f07b78/kingdom"
```` 
be
```json
["https:", "", "", "mangadex.org", "title", "077a3fed-1634-424f-be7a-9a96b7f07b78", "kingdom"]
```

And the script grabs the last item, which in this case is `"kingdom"`.
And you can clearly tell that that is not its id.
I fix this bug in my PR, so when it's merged this problem will go away.

**Comment by orbea** [comment]
Link to PR for future reference: https://github.com/frozenpandaman/mangadex-dl/pull/39

**Comment by frozenpandaman** [comment]
Fixed in 222789e – the script should now be able to accept any URL structure, with the name of the manga as a slug on the end or not. Thank you!
